pkg/api/k8s_versions.go

```diff
@@ -30,7 +30,7 @@ var k8sComponentVersions = map[string]map[string]string{
 		"aci-connector":                    "virtual-kubelet:latest",
 		ContainerMonitoringAddonName:       "oms:ciprod10162018-2",
 		AzureCNINetworkMonitoringAddonName: "networkmonitor:v0.0.4",
-		"cluster-autoscaler":               "cluster-autoscaler:v1.3.3",
+		"cluster-autoscaler":               "cluster-autoscaler:v1.12.1",
```

Comment:
Is there a good reason for including this change in this PR?

Reply:
That seems incorrect then. Should be 1.12.x for 1.12.x and 1.13.x for 1.13... I'll fix it up.

Reply:
I fixed this--thanks for noticing! `cluster-autoscaler` v1.12.1 appears to be the current 1.12.x release, so I left it there.

Reply:
See also #170.